README.md

```diff
@@ -110,3 +110,8 @@ public class DemoApp {
 ```
 
 For more examples see [pg-index-health-demo](https://github.com/mfvanek/pg-index-health-demo) project.
+
+## Questions, issues, feature requests, and contributions
```

Comment:
It looks like there is unnecessary comma before "and"

Reply:
This was intended, see https://en.wikipedia.org/wiki/Serial_comma. However, I've removed it.